src/GitTfs/Core/GitTfsRemote.cs

```diff
@@ -958,18 +958,18 @@ private void PendChangesToWorkspace(string head, string parent, ITfsWorkspaceMod
         public int Checkin(string head, TfsChangesetInfo parentChangeset, CheckinOptions options, string sourceTfsPath = null)
         {
             var changeset = 0;
-            WithWorkspace(parentChangeset, workspace => changeset = Checkin(head, parentChangeset.GitCommit, workspace, options, sourceTfsPath));
+            WithWorkspace(parentChangeset, workspace => changeset = Checkin(head, parentChangeset.GitCommit, workspace, options, sourceTfsPath), options.WorkspaceOwner);
             return changeset;
         }
 
         public int Checkin(string head, string parent, TfsChangesetInfo parentChangeset, CheckinOptions options, string sourceTfsPath = null)
         {
             var changeset = 0;
-            WithWorkspace(parentChangeset, workspace => changeset = Checkin(head, parent, workspace, options, sourceTfsPath));
+            WithWorkspace(parentChangeset, workspace => changeset = Checkin(head, parent, workspace, options, sourceTfsPath), options.WorkspaceOwner);
             return changeset;
         }
 
-        private void WithWorkspace(TfsChangesetInfo parentChangeset, Action<ITfsWorkspace> action)
+        private void WithWorkspace(TfsChangesetInfo parentChangeset, Action<ITfsWorkspace> action, string workspaceOwner = "")
```

Comment:
don't make workspaceOwner an optional arg, shouldn't be needed.

Reply:
I agree with you! I fixed this in the 5e98626